client/src/routes/widgetModule/WidgetModule.tsx

```diff
@@ -95,6 +125,188 @@ export const WidgetModule: React.FC<Props> = observer(({ domainName }) => {
     setFormFields({ ...formFields, widgetValue: '' })
   }
 
+  const createAlias = async (alias: string, forward: string) => {
+    setProcessStatus({
+      type: ProcessStatusTypes.PROGRESS,
+      render: (
+        <BaseText>
+          {walletStore.isMetamaskAvailable
+            ? 'Waiting for a transaction to be signed'
+            : 'Sign transaction on mobile device'}
+        </BaseText>
+      ),
+    })
+
+    try {
+      const numAlias = await rootStore.easClient.getNumAlias(domainName)
+      const maxAlias = await rootStore.easClient.maxNumAlias()
+      const publicAliases = await rootStore.easClient.getPublicAliases(
+        domainName
+      )
+      console.log('### maxNum', numAlias)
+      console.log('### numAlias', numAlias)
+      console.log('### publicAliases', publicAliases)
+
+      if (numAlias >= maxAlias) {
+        setProcessStatus({
+          type: ProcessStatusTypes.PROGRESS,
+          render: (
+            <BaseText>
+              {walletStore.isMetamaskAvailable
+                ? 'Waiting for a transaction to be signed'
+                : 'Sign transaction on mobile device'}
+            </BaseText>
+          ),
+        })
+
+        const removingAlias = publicAliases[0]
```

Comment:
why?

Reply:
the issue: when an user try to add alias and he have one already we should replace it
the reason: at the moment an user encounter with max limit = 1

if you have suggesting with another approach, i would be pleased

Reply:
> Was it tested end-to-end?

yes, i tested it

Reply:
you can check if alias is the same (e.g. hello), if so, just call activate on both contract and server, it will update